save.go

```diff
@@ -93,6 +93,12 @@ func (d *InMAP) initFromCells(cells []*Cell, emis *Emissions, config *VarGridCon
 	for i, p := range config.CensusPopColumns {
 		d.popIndices[p] = i
 	}
+	d.mortIndices = make(map[string]int)
+	i := 0
+	for _, m := range config.MortalityRateColumns {
```

Comment:
I wrote it this way because `for i, m := range config.MortalityRateColumns {` would make `i` loop through the keys in the map. I needed to instead count up from zero in order to have an index for each mortality rate for assignment in the `d.mortIndices` map.